content/kubermatic/main/architecture/supported-providers/kubevirt/_index.en.md

```diff
@@ -14,37 +14,26 @@ weight = 5
 ### Requirements
 
 A Kubernetes cluster (KubeVirt infrastructure cluster), which consists of nodes that **have a hardware virtualization support** with at least:
-* 2 CPUs
-* 4GB of RAM
-* 30GB of storage.
+* 3 Bare Metal Server
+* CPUs: Minimum 8-core for testing; minimum 16-core or more for production
+* Memory: Minimum 32 GB for testing; minimum 64 GB or more for production
+* Storage: Minimum 100 GB for testing; minimum 500 GB or more for production
 
-The cluster version must be in the scope of [supported KKP Kubernetes clusters]({{< ref "../../../tutorials-howtos/operating-system-manager/compatibility/#kubernetes-versions" >}})
-and it must have the following components installed:
-* KubeVirt >= 0.57 which supports the selected Kubernetes version.
-* Containerized Data Importer which supports the selected KubeVirt and Kubernetes versions.
-
-{{% notice note %}}
-We recommend to install the latest stable releases of both projects.
-{{% /notice %}}
-
-The setup has been successfully tested with:
-* CRI: containerd
-* CNI: Canal
+Software requirement:
+* KubeOne with Canal = 1.7 or higher
```

Comment:
This is not a requirement and it can be a bit misleading.

Reply:
I would say we should drop it and only say Kubernetes and stick with this k8s requirements https://kubevirt.io/user-guide/cluster_admin/installation/#requirements